### DISCUSSION TITLE
Linode or Azure deployment problem

### DISCUSSION THREAD

**Discussion by cpalop** [AzuraCast/AzuraCast#4734] [General]
I've deployed azuracast on both platforms but i'm getting this error while creating the account

<img width="480" alt="Captura de pantalla 2021-10-27 a las 13 42 09" src="https://user-images.githubusercontent.com/20813397/139059009-3224d033-99d0-4306-838b-8ecb96745b7f.png">

not sure whats wrong

**Comment by BusterNeece** [comment]
@cpalop This is an issue with our latest rolling-release version and has since been fixed; if you follow the instructions to update your installation, you should be able to proceed through the initial setup without any issue. Sorry for any inconvenience!

**Comment by ryanginnelly252** [comment]
Hi, just to verify today I created an instance on Linode. Everything seems fine.
